# Baltimore Classification

**Baltimore classification** is a system used to
[classify](Virus_classification "wikilink") [viruses](virus "wikilink")
based on their manner of [messenger RNA](messenger_RNA "wikilink")
(mRNA) synthesis. By organizing viruses based on their manner of mRNA
production, it is possible to study viruses that behave similarly as a
distinct group. Seven Baltimore groups are described that take into
consideration whether the viral genome is made of [deoxyribonucleic
acid](deoxyribonucleic_acid "wikilink") (DNA) or [ribonucleic
acid](ribonucleic_acid "wikilink") (RNA), whether the genome is single-
or double-stranded, and whether the
[sense](Sense_(molecular_biology)#RNA_sense_in_viruses "wikilink") of a
single-stranded RNA genome is positive or negative.

Baltimore classification also closely corresponds to the manner of
[replicating](Viral_replication "wikilink") the genome, so Baltimore
classification is useful for grouping viruses together for both
[transcription](Transcription_(biology) "wikilink") and replication.
Certain subjects pertaining to viruses are associated with multiple,
specific Baltimore groups, such as specific forms of
[translation](Translation_(biology) "wikilink") of mRNA and the host
range of different types of viruses. Structural characteristics such as
the shape of the viral [capsid](capsid "wikilink"), which stores the
viral genome, and the evolutionary history of viruses are not
necessarily related to Baltimore groups.

Baltimore classification was created in 1971 by virologist [David
Baltimore](David_Baltimore "wikilink"). Since then, it has become common
among virologists to use Baltimore classification alongside standard
virus taxonomy, which is based on evolutionary history. In 2018 and
2019, Baltimore classification was partially integrated into virus
taxonomy based on evidence that certain groups were descended from
common ancestors. Various realms, kingdoms, and phyla now correspond to
specific Baltimore groups.

## Overview

Baltimore classification groups viruses together based on their manner
of mRNA synthesis. Characteristics directly related to this include
whether the genome is made of [deoxyribonucleic
acid](deoxyribonucleic_acid "wikilink") (DNA) or [ribonucleic
acid](ribonucleic_acid "wikilink") (RNA), the strandedness of the
genome, which can be either single- or double-stranded, and the sense of
a single-stranded genome, which is either positive or negative. The
primary advantage of Baltimore classification is that by classifying
viruses according to the aforementioned characteristics, viruses that
behave in the same manner can be studied as distinct groups. There are
seven Baltimore groups numbered with Roman numerals, listed hereafter.

-   Group I: double-stranded DNA viruses
-   Group II: single-stranded DNA viruses
-   Group III: double-stranded RNA viruses
-   Group IV: positive sense single-stranded RNA viruses
-   Group V: negative sense single-stranded RNA viruses
-   Group VI: single-stranded RNA viruses with a DNA intermediate in
    their life cycle
-   Group VII: double-stranded DNA viruses with an RNA intermediate in
    their life cycle

Baltimore classification is chiefly based on the
[transcription](Transcription_(biology) "wikilink") of the viral genome,
and viruses within each group typically share the manners by which the
mRNA synthesis occurs. While not the direct focus of Baltimore
classification, groups are organized in such a manner that viruses in
each group also typically have the same mechanisms of
[replicating](Viral_replication "wikilink") the viral genome.[1] Because
of this, Baltimore classification provides insights into both the
transcription and replication parts of the [viral life
cycle](viral_life_cycle "wikilink"). Structural characteristics of a
virus particle, called a virion, such as the shape of the viral capsid
and the presence of a [viral envelope](viral_envelope "wikilink"), a
[lipid](lipid "wikilink") membrane that surrounds the capsid, have no
direct relation to Baltimore groups, nor do the groups necessarily show
genetic relation based on evolutionary history.

## Classification

### DNA viruses

DNA viruses have genomes made of deoxyribonucleic acid (DNA) and are
organized into two groups: double-stranded DNA (dsDNA) viruses, and
single-stranded DNA (ssDNA) viruses. They are assigned to four separate
realms: *[Adnaviria](Adnaviria "wikilink")*,
*[Duplodnaviria](Duplodnaviria "wikilink")*,
*[Monodnaviria](Monodnaviria "wikilink")*, and
*[Varidnaviria](Varidnaviria "wikilink")*. Many have yet to be assigned
to a realm.

#### Group I: double-stranded DNA viruses

The first Baltimore group contains viruses that have a double-stranded
DNA (dsDNA) genome. All dsDNA viruses have their mRNA synthesized in a
three-step process. First, a [transcription preinitiation
complex](transcription_preinitiation_complex "wikilink") binds to the
DNA upstream of the site where transcription begins, allowing for the
recruitment of a host [RNA polymerase](RNA_polymerase "wikilink").
Second, once the RNA polymerase is recruited, it uses the negative
strand as a template for synthesizing mRNA strands. Third, the RNA
polymerase terminates transcription upon reaching a specific signal,
such as a [polyadenylation](polyadenylation "wikilink") site.[2]

dsDNA viruses make use of several mechanisms to replicate their genome.
Bidirectional replication, in which two replication forks are
established at a replication origin site and move in opposite directions
of each other, is widely used.[3] A rolling circle mechanism that
produces linear strands while progressing in a loop around the circular
genome is also common.[4] Some dsDNA viruses use a strand displacement
method whereby one strand is synthesized from a template strand, and a
complementary strand is then synthesized from the prior synthesized
strand, forming a dsDNA genome.[5] Lastly, some dsDNA viruses are
replicated as part of a process called [replicative
transposition](replicative_transposition "wikilink") whereby a viral
genome in a host cell's DNA is replicated to another part of a host
genome.[6]

dsDNA viruses can be subdivided between those that replicate in the
nucleus, and as such are relatively dependent on host cell machinery for
transcription and replication, and those that replicate in the
cytoplasm, in which case they have evolved or acquired their own means
of executing transcription and replication. dsDNA viruses are also
commonly divided between tailed dsDNA viruses, referring to members of
the realm *Duplodnaviria*, usually the tailed bacteriophages of the
order *Caudovirales*, and tailless or non-tailed dsDNA viruses of the
realm *Varidnaviria*.[7][8]

dsDNA viruses are classified into three of the four realms and include
many taxa that are unassigned to a realm:

-   All viruses in *Adnaviria* are dsDNA viruses. Viruses in this realm
    infect [archaea](archaea "wikilink").[9]
-   All viruses in *Duplodnaviria* are dsDNA viruses. Viruses in this
    realm belong to two groups: tailed bacteriophages in
    *[Caudovirales](Caudovirales "wikilink")* and herpesviruses in
    *[Herpesvirales](Herpesvirales "wikilink")*.[10]
-   In *Monodnaviria*, members of the class
    *[Papovaviricetes](Papovaviricetes "wikilink")* are dsDNA viruses.
    Viruses in *Papovaviricetes* constitute two groups:
    [papillomaviruses](papillomavirus "wikilink") and
    [polyomaviruses](polyomavirus "wikilink").[11]
-   All viruses in *Varidnaviria* are dsDNA viruses. Viruses in this
    realm include [adenoviruses](adenovirus "wikilink"), [giant
    viruses](giant_virus "wikilink"), and
    [poxviruses](poxvirus "wikilink").[12]
-   The following taxa that are unassigned to a realm exclusively
    contain dsDNA viruses:[13]
    -   Classes: *[Naldaviricetes](Naldaviricetes "wikilink")*[14]
    -   Families: *[Ampullaviridae](Ampullaviridae "wikilink")*,
        *[Bicaudaviridae](Bicaudaviridae "wikilink")*,
        *[Clavaviridae](Clavaviridae "wikilink")*,
        *[Fuselloviridae](Fuselloviridae "wikilink")*,
        *[Globuloviridae](Globuloviridae "wikilink")*,
        *[Guttaviridae](Guttaviridae "wikilink")*,
        *[Halspiviridae](Halspiviridae "wikilink")*,
        *[Ovaliviridae](Ovaliviridae "wikilink")*,
        *[Plasmaviridae](Plasmaviridae "wikilink")*,
        *[Polydnaviridae](Polydnaviridae "wikilink")*,
        *[Portogloboviridae](Portogloboviridae "wikilink")*,
        *[Thaspiviridae](Thaspiviridae "wikilink")*
    -   Genera: *[Dinodnavirus](Dinodnavirus "wikilink")*,
        *[Rhizidiovirus](Rhizidiovirus "wikilink")*

#### Group II: single-stranded DNA viruses

The second Baltimore group contains viruses that have a single-stranded
DNA (ssDNA) genome. ssDNA viruses have the same manner of transcription
as dsDNA viruses. Because the genome is single-stranded, however, it is
first made into a double-stranded form by a [DNA
polymerase](DNA_polymerase "wikilink") upon entering a host cell. mRNA
is then synthesized from the double-stranded form. The double-stranded
form of ssDNA viruses may be produced either directly after entry into a
cell or as a consequence of replication of the viral genome.[15][16]
Eukaryotic ssDNA viruses are replicated in the nucleus.

Most ssDNA viruses contain circular genomes that are replicated via
rolling circle replication (RCR). ssDNA RCR is initiated by an
[endonuclease](endonuclease "wikilink") that bonds to and cleaves the
positive strand, allowing a DNA polymerase to use the negative strand as
a template for replication. Replication progresses in a loop around the
genome by means of extending the 3â€²-end of the positive strand,
displacing the prior positive strand, and the endonuclease cleaves the
positive strand again to create a standalone genome that is
[ligated](Ligation_(molecular_biology) "wikilink") into a circular loop.
The new ssDNA may be packaged into virions or replicated by a DNA
polymerase to form a double-stranded form for transcription or
continuation of the replication cycle.[17]

[Parvoviruses](Parvovirus "wikilink") contain linear ssDNA genomes that
are replicated via [rolling hairpin
replication](rolling_hairpin_replication "wikilink") (RHR), which is
similar to RCR. Parvovirus genomes have [hairpin
loops](Stem-loop "wikilink") at each end of the genome that repeatedly
unfold and refold during replication to change the direction of DNA
synthesis to move back and forth along the genome, producing numerous
copies of the genome in a continuous process. Individual genomes are
then excised from this molecule by the viral endonuclease. For
parvoviruses, either the positive or negative sense strand may be
packaged into capsids, varying from virus to virus.[18]

Nearly all ssDNA viruses have positive sense genomes, but a few
exceptions and peculiarities exist. The family
*[Anelloviridae](Anelloviridae "wikilink")* is the only ssDNA family
whose members have negative sense genomes, which are circular.
Parvoviruses, as previously mentioned, may package either the positive
or negative sense strand into virions.[19] Lastly,
[bidnaviruses](Bidensovirus "wikilink") package both the positive and
negative linear strands.[20] In any case, the sense of ssDNA viruses,
unlike for ssRNA viruses, is not sufficient to separate ssDNA viruses
into two groups since all ssDNA viral genomes are converted to dsDNA
forms prior to transcription and replication.[21]

ssDNA viruses are classified into one of the four realms and include
several families that are unassigned to a realm:

-   In *Monodnaviria*, all members except viruses in *Papovaviricetes*
    are ssDNA viruses.[22]
-   The unassigned families *[Anelloviridae](Anelloviridae "wikilink")*
    and *[Spiraviridae](Spiraviridae "wikilink")* are ssDNA virus
    families.[23]
-   Viruses in the family
    *[Finnlakeviridae](Finnlakeviridae "wikilink")* contain ssDNA
    genomes. *Finnlakeviridae* is unassigned to a realm but is a
    proposed member of *Varidnaviria*.[24]

### RNA viruses

RNA viruses have genomes made of ribonucleic acid (RNA) and comprise
three groups: double-stranded RNA (dsRNA) viruses, positive sense
single-stranded RNA (+ssRNA) viruses, and negative sense single-stranded
RNA (-ssRNA) viruses. The majority of RNA viruses are classified in the
kingdom *[Orthornavirae](Orthornavirae "wikilink")* in the realm
*[Riboviria](Riboviria "wikilink")*. The exceptions are generally
[viroids](viroid "wikilink") and other [subviral
agents](subviral_agent "wikilink"). Some of the latter category, such as
the [hepatitis D virus](hepatitis_D_virus "wikilink"), are classified in
the realm *[Ribozyviria](Ribozyviria "wikilink")*.

#### Group III: double-stranded RNA viruses

The third Baltimore group contains viruses that have a double-stranded
RNA (dsRNA) genome. After entering a host cell, the dsRNA genome is
transcribed to mRNA from the negative strand by the viral RNA-dependent
RNA polymerase (RdRp). The mRNA may be used for translation or
replication. Single-stranded mRNA is replicated to form the dsRNA
genome. The 5â€²-end of the genome may be naked, capped, or covalently
bound to a viral protein.[25]

dsRNA is not a molecule made by cells, so cellular life has evolved
antiviral systems to detect and inactivate viral dsRNA. To counteract
this, many dsRNA genomes are constructed inside of capsids, thereby
avoiding detection inside of the host cell's cytoplasm. mRNA is forced
out from the capsid in order to be translated or to be translocated from
a mature capsid to a progeny capsid.[26] While dsRNA viruses typically
have capsids, viruses in the families
*[Amalgaviridae](Amalgaviridae "wikilink")* and
*[Endornaviridae](Endornaviridae "wikilink")* have not been observed to
form virions and as such apparently lack capsids. Endornaviruses are
also unusual in that unlike other RNA viruses, they possess a single,
long [open reading frame](open_reading_frame "wikilink") (ORF), or
translatable portion, and a site-specific nick in the 5â€² region of the
positive strand.

dsRNA viruses are classified into two phyla within the kingdom
*Orthornavirae* of the realm *Riboviria*:[27]

-   All viruses in *[Duplornaviricota](Duplornaviricota "wikilink")* are
    dsRNA viruses.
-   In *[Pisuviricota](Pisuviricota "wikilink")*, all members of the
    class *[Duplopiviricetes](Duplopiviricetes "wikilink")* are dsRNA
    viruses.

#### Group IV: positive sense single-stranded RNA viruses

The fourth Baltimore group contains viruses that have a positive sense
single-stranded RNA (+ssRNA) genome. For +ssRNA viruses, the genome
functions as mRNA, so no transcription is required for translation.
+ssRNA viruses will also, however, produce positive sense copies of the
genome from negative sense strands of an intermediate dsRNA genome. This
acts as both a transcription and a replication process since the
replicated RNA is also mRNA. The 5â€²-end may be naked, capped, or
covalently bound to a viral protein, and the 3â€²-end may be naked or
polyadenylated.[28]

Many +ssRNA viruses are able to have only a portion of their genome
transcribed. Typically, subgenomic RNA (sgRNA) strands are used for
translation of structural and movement proteins needed during
intermediate and late stages of infection. sgRNA transcription may occur
by commencing RNA synthesis within the genome rather than from the
5â€²-end, by stopping RNA synthesis at specific sequences in the genome,
or by, as a part of both prior methods, synthesizing [leader
sequences](Five_prime_untranslated_region "wikilink") from the viral RNA
that are then attached to sgRNA strands. Because replication is required
for sgRNA synthesis, RdRp is always translated first.[29]

Because the process of replicating the viral genome produces
intermediate dsRNA molecules, +ssRNA viruses can be targeted by the host
cell's immune system. To avoid detection, +ssRNA viruses replicate in
membrane-associated vesicles that are used as replication factories.
From there, only viral +ssRNA, which may be mRNA, enters the main
cytoplasmic area of the cell.[30]

+ssRNA viruses can be subdivided between those that have polycistronic
mRNA, which encodes a polyprotein that is cleaved to form multiple
mature proteins, and those that produce subgenomic mRNAs and therefore
undergo two or more rounds of translation. +ssRNA viruses are included
in three phyla in the kingdom *Orthornavirae* in the realm
*Riboviria*:[31]

-   All viruses in *[Lenarviricota](Lenarviricota "wikilink")* are
    +ssRNA viruses.
-   All viruses in *[Pisuviricota](Pisuviricota "wikilink")* are +ssRNA
    viruses, excluding the class
    *[Duplopiviricetes](Duplopiviricetes "wikilink")*, whose members
    have dsRNA genomes.
-   All viruses in *[Kitrinoviricota](Kitrinoviricota "wikilink")* are
    +ssRNA viruses.

#### Group V: negative sense single-stranded RNA viruses

The fifth Baltimore group contains viruses that have a negative sense,
single-stranded RNA (-ssRNA) genome. mRNA, which is positive sense, is
transcribed directly from the negative sense genome. The first process
for -ssRNA transcription involves RdRp binding to a leader sequence on
the 3â€² end of the genome, transcribing a 5â€² triphosphate-leader RNA that
is capped, then stopping and restarting on a transcription signal which
is [capped](Five-prime_cap "wikilink"), continuing until a stop signal
is reached.[32] The second manner is similar but instead of synthesizing
a cap, RdRp may make use of [cap snatching](cap_snatching "wikilink"),
whereby a short sequence of host cell mRNA is taken and used as the 5â€²
cap of the viral mRNA.[33] Genomic -ssRNA is replicated from the
positive sense antigenome in a similar manner as transcription, except
in reverse using the antigenome as a template for the genome. RdRp moves
from the 3â€²-end to the 5â€²-end of the antigenome and ignores all
transcription signals when synthesizing genomic -ssRNA.[34]

Various -ssRNA viruses use special mechanisms for transcription. The
manner of producing the polyA tail may be via [polymerase
stuttering](polymerase_stuttering "wikilink"), during which RdRp
transcribes an [adenine](adenine "wikilink") from
[uracil](uracil "wikilink") and then moves back in the RNA sequence with
the mRNA to transcribe it again, continuing this process numerous times
until hundreds of adenines have been added to the 3â€²-end of the
mRNA.[35] Additionally, some -ssRNA viruses are ambisense, as both the
positive and negative strands separately encode viral proteins, and
these viruses produce two separate mRNA strands: one directly from the
genome and one from a complementary strand.[36]

-ssRNA viruses can be subdivided informally between those that have
nonsegmented and segmented genomes. Nonsegmented -ssRNA viruses
replicate in the cytoplasm, and segmented -ssRNA viruses replicate in
the nucleus. During transcription, the RdRp produces one monocistronic
mRNA strand from each segment of the genome. All -ssRNA viruses are
classified in the phylum *[Negarnaviricota](Negarnaviricota "wikilink")*
in the kingdom *Orthornavirae* in the realm *Riboviria*.
*Negarnaviricota* only contains -ssRNA viruses, so "-ssRNA virus" is
synonymous with *Negarnaviricota*.[37] *Negarnaviricota* is divided into
two subphyla: *Haploviricotina*, whose members synthesize a cap
structure on viral mRNA required for protein synthesis, and
*Polyploviricotina*, whose members instead obtain caps on mRNA via cap
snatching.[38]

### Reverse transcribing viruses

Reverse transcribing (RT) viruses have genomes made of either DNA or RNA
and replicate via reverse transcription. Two groups of reverse
transcribing viruses exist: single-stranded RNA-RT (ssRNA-RT) viruses,
and double-stranded DNA-RT (dsDNA-RT) viruses. Reverse transcribing
viruses are classified in the kingdom *Pararnavirae* in the realm
*Riboviria*.

#### Group VI: single-stranded RNA viruses with a DNA intermediate

The sixth Baltimore group contains viruses that have a (positive-sense)
single-stranded RNA genome that has a DNA intermediate ((+)ssRNA-RT) in
its replication cycle.[39] ssRNA-RT viruses are transcribed in the same
manner as DNA viruses, but their linear genomes are first converted to a
dsDNA form through a process called [reverse
transcription](reverse_transcription "wikilink"). The viral [reverse
transcriptase](reverse_transcriptase "wikilink") enzyme synthesizes a
DNA strand from the ssRNA strand, and the RNA strand is degraded and
replaced with a DNA strand to create a dsDNA genome. The genome is then
[integrated](integrase "wikilink") into the DNA of the host cell, where
it is now called a [provirus](provirus "wikilink"). The host cell's [RNA
polymerase II](RNA_polymerase_II "wikilink") then transcribes RNA in the
nucleus from the proviral DNA. Some of this RNA may become mRNA whereas
other strands will become copies of the viral genome for
replication.[40]

ssRNA-RT viruses are all included in the class
*[Revtraviricetes](Revtraviricetes "wikilink")*, phylum *Arterviricota*,
kingdom *Pararnavirae* of the realm *Riboviria*. Excluding
*Caulimoviridae*, which belongs to Group VII, all members of the
*Revtraviricetes* order *[Ortervirales](Ortervirales "wikilink")* are
ssRNA-RT viruses.[41][42]

#### Group VII: double-stranded DNA viruses with an RNA intermediate

The seventh Baltimore group contains viruses that have a double-stranded
DNA genome that has an RNA intermediate (dsDNA-RT) in its replication
cycle. dsDNA-RT viruses have a gap in one strand, which is
[repaired](DNA_repair "wikilink") to create a complete dsDNA genome
prior to transcription. dsDNA-RT viruses are transcribed in the same
manner as dsDNA viruses,[43] but make use of reverse transcription to
replicate their circular genome while it is still in the capsid. The
host cell's RNA polymerase II transcribes RNA strands from the genome in
the cytoplasm, and the genome is replicated from these RNA strands. The
dsDNA genome is produced from pregenomic RNA strands via the same
general mechanism as ssRNA-RT viruses, but with replication occurring in
a loop around the circular genome. After replication, the dsDNA genome
may be packed or sent to the nucleus for further rounds of
transcription.[44]

dsDNA-RT viruses are, like ssRNA-RT, all included in the class
*Revtraviricetes*. Two families of dsDNA-RT viruses are recognized:
*[Caulimoviridae](Caulimoviridae "wikilink")*, which belongs to the
order *Ortervirales*, and *[Hepadnaviridae](Hepadnaviridae "wikilink")*,
which is the sole family in the order *Blubervirales*.[45][46]

## Multi-group characteristics

A number of characteristics of viruses are not directly associated with
Baltimore classification but nonetheless closely correspond to multiple,
specific Baltimore groups. This includes alternative splicing during
transcription, whether the viral genome is segmented, the host range of
viruses, whether the genome is linear or circular, and different methods
of translating viral mRNA.

### Alternative splicing

[Alternative splicing](Alternative_splicing "wikilink") is a mechanism
by which different proteins can be produced from a single gene by means
of using alternative splicing sites to produce different mRNAs. It is
found in various DNA, -ssRNA, and reverse transcribing viruses. Viruses
may make use of alternative splicing solely to produce multiple proteins
from a single pre-mRNA strand or for other specific purposes. For
certain viruses, including the families
*[Orthomyxoviridae](Orthomyxoviridae "wikilink")* and
*[Papillomaviridae](Papillomaviridae "wikilink")*, alternative splicing
acts as a way to regulate early and late [gene
expression](gene_expression "wikilink") during different stages of
infection. [Herpesviruses](Herpesvirus "wikilink") use it as a potential
anti-host defense mechanism to prevent synthesis of specific antiviral
proteins. Furthermore, in addition to alternative splicing, because
cellular [unspliced RNA](RNA_splicing "wikilink") cannot be transported
out of the nucleus, hepadnaviruses and
[retroviruses](retrovirus "wikilink") contain their own proteins for
exporting their unspliced genomic RNA out of the nucleus.[47]

### Genome segmentation

Viral genomes can exist in a single, or monopartite, segment, or they
may be split into more than one molecule, called multipartite. For
monopartite viruses, all genes are on the single segment of the genome.
Multipartite viruses typically package their genomes into a single
virion so that the whole genome is in one virus particle, and the
separate segments contain different genes. Monopartite viruses are found
in all Baltimore groups, whereas multipartite viruses are usually RNA
viruses. This is because most multipartite viruses infect plants or
fungi, which are eukaryotes, and most eukaryotic viruses are RNA
viruses.[48][49] The family
*[Pleolipoviridae](Pleolipoviridae "wikilink")* varies as some viruses
are monopartite ssDNA while others are bipartite with one segment being
ssDNA and the other dsDNA.[50] Viruses in the ssDNA [plant
virus](plant_virus "wikilink") family
*[Geminiviridae](Geminiviridae "wikilink")* likewise vary between being
monopartite and bipartite.[51]

### Host range

Different Baltimore groups tend to be found within different branches of
cellular life. In prokaryotes, the large majority of viruses are dsDNA
viruses, and a significant minority are ssDNA viruses. Prokaryotic RNA
viruses, in contrast, are relatively rare. Most eukaryotic viruses,
including most animal and plant viruses, are RNA viruses, although
eukaryotic DNA viruses are also common. By group, the vast majority of
dsDNA viruses infect prokaryotes, ssDNA viruses are found in all three
domains of life, dsRNA and +ssRNA viruses are primarily found in
eukaryotes but also in bacteria, and -ssRNA and reverse transcribing
viruses are only found in eukaryotes.[52][53]

### Linear vs circular genomes

Viral genomes may be either linear with ends or circular in a loop.
Whether a virus has a linear or circular genome varies from group to
group. A significant percentage of dsDNA viruses are both, ssDNA viruses
are primarily circular, RNA viruses and ssRNA-RT viruses are typically
linear, and dsDNA-RT viruses are typically circular.[54][55] In the
dsDNA family *[Sphaerolipoviridae](Sphaerolipoviridae "wikilink")*, and
in the family *Pleolipoviridae*, viruses contain both linear and
circular genomes, varying from genus to genus.[56][57]

### RNA editing

RNA editing is used by various ssRNA viruses to produce different
proteins from a single gene. This can be done via polymerase slippage
during transcription or by post-transcriptional editing. In polymerase
slippage, the RNA polymerase slips one nucleotide back during
transcription, inserting a nucleotide not included in the template
strand. Editing of a genomic template would impair gene expression, so
RNA editing is only done during and after transcription. For [ebola
viruses](Ebolavirus "wikilink"), RNA editing improves the ability to
adapt to their hosts.[58]

Alternative splicing differs from RNA editing in that alternative
splicing does not change the mRNA sequence like RNA editing but instead
changes the coding capacity of an mRNA sequence as a result of
alternative splicing sites. The two mechanisms otherwise have the same
result: multiple proteins are expressed from a single gene.

### Translation

[Translation](Translation_(biology) "wikilink") is the process by which
[proteins](protein "wikilink") are synthesized from mRNA by
[ribosomes](ribosome "wikilink"). Baltimore groups do not directly
pertain to the translation of viral proteins, but various atypical types
of translation used by viruses are usually found within specific
Baltimore groups:[59][60]

-   Non-canonical translation initiation:
    -   Viral initiation of translation: used primarily by +ssRNA and
        ssRNA-RT viruses, various viruses have evolved mechanisms to
        initiate translation, such as having internal ribosomal entry
        sites to allow for cap-independent translation, having
        downstream hairpin loops that allow for cap-dependent
        translation in the absence of an [eIF2](eIF2 "wikilink")
        initiation factor, and initiation at a CUG or other start codon
        with a [leucine](leucine "wikilink") amino acid.[61]
    -   [Leaky scanning](Leaky_scanning "wikilink"): used by various
        viruses in all Baltimore groups, the 40S ribosomal subunit may
        scan through a start codon, thereby skipping an ORF, only
        initiating translation with the 60S subunit at a subsequent
        start codon.[62]
    -   [Ribosomal shunting](Ribosomal_shunting "wikilink"): used by
        various dsDNA, +ssRNA, -ssRNA, ssRNA-RT, a dsDNA-RT viruses,
        ribosomes will start scanning from a 5â€²-cap structure then
        bypass a leader structure in the mRNA, initiation translation
        downstream from the leader sequence.[63]
    -   Termination-reinitiation: used by some dsRNA and +ssRNA viruses,
        ribosomes may translate an ORF, but following termination of
        translation of that ORF, a proportion of [40S subunits of the
        ribosome](Eukaryotic_small_ribosomal_subunit_(40S) "wikilink")
        remain attached to the mRNA as a way to reinitiate translation
        of a subsequent ORF.[64]
-   Non-canonical elongation and termination of translation:
    -   [Ribosomal frameshifting](Ribosomal_frameshifting "wikilink"):
        used by various dsDNA, dsRNA, +ssRNA, and ssRNA-RT viruses,
        produces merged proteins from overlapping ORFs. This is executed
        simply by ribosomes slipping one
        [nucleobase](nucleobase "wikilink") forward or backward during
        translation.[65]
    -   Suppression of termination: also called stop-codon readthrough,
        used by various dsRNA, +ssRNA, and ssRNA-RT viruses, certain
        viruses contain codons in their mRNA that would normally signal
        for termination of translation upon being recognized by a
        [release factor](release_factor "wikilink") but are instead
        partially recognized by [tRNA](tRNA "wikilink") during
        translation, which allows for continued translation up to the
        next stop codon in order to produce an extended end of the viral
        protein.[66] In viruses, this is often used to express
        [replicase](replicase "wikilink") enzymes.
    -   Ribosomal skipping: also called stop-carry on, used by various
        dsRNA and +ssRNA viruses, a viral peptide, or amino acid
        sequence, may prevent a ribosome from covalently linking a new
        inserted amino acid, which blocks further translation.
        Consequently, the polyprotein is co-translationally cleaved, and
        a new amino acid sequence is started, leading to the production
        of two individual proteins from one ORF.[67]

## History

Baltimore classification was proposed in 1971 by virologist [David
Baltimore](David_Baltimore "wikilink") in a paper titled *Expression of
Animal Virus Genomes*. It initially contained the first six groups but
was later expanded to include group VII.[68][69] Because of the utility
of Baltimore classification, it has come to be used alongside standard
virus taxonomy, which is based on evolutionary relationships and
governed by the [International Committee on Taxonomy of
Viruses](International_Committee_on_Taxonomy_of_Viruses "wikilink")
(ICTV).[70]

From the 1990s to the 2010s, virus taxonomy used a 5-rank system ranging
from order to species with Baltimore classification used in conjunction.
Outside of the ICTV's official framework, various supergroups of viruses
joining together different families and orders were created over time
based on increasing evidence of deeper evolutionary relations.
Consequently, in 2016, the ICTV began to consider establishing ranks
higher than order as well as how the Baltimore groups would be treated
among higher taxa.[71]

In two votes in 2018 and 2019, a 15-rank system ranging from
[realm](Realm_(virology) "wikilink") to species was established by the
ICTV.[72] As part of this, the Baltimore groups for RNA viruses and RT
viruses were incorporated into formal taxa. In 2018, the realm
*Riboviria* was established and initially included the three RNA virus
groups.[73] A year later, *Riboviria* was expanded to also include both
RT groups. Within the realm, RT viruses are included in the kingdom
*Pararnavirae* and RNA viruses in the kingdom *Orthornavirae*.
Furthermore, the three Baltimore groups for RNA viruses are used as
defining characteristics of the phyla in *Orthornavirae*.[74]

Unlike RNA viruses and RT viruses, DNA viruses have not been united
under a single realm but are instead dispersed across four realms and
various taxa that are not assigned to a realm. The realms *Adnaviria*
and *Duplodnaviria* exclusively contains dsDNA viruses,[75][76]
*Monodnaviria* primarily contains ssDNA viruses but also contains dsDNA
viruses,[77] and *Varidnaviria* exclusively contains dsDNA viruses,
although some proposed members of *Varidnaviria*, namely the family
*[Finnlakeviridae](Finnlakeviridae "wikilink")*, are ssDNA viruses.[78]